src/deadline/client/cli/_groups/bundle_group.py

```diff
@@ -98,9 +98,14 @@ def validate_parameters(ctx, param, value):
     type=click.Choice([e.value for e in JobAttachmentsFileSystem]),
 )
 @click.option(
-    "--yes",
+    "--upload",
     is_flag=True,
-    help="Skip any confirmation prompts",
+    help="Indicates that files will be uploaded to S3 if needed",
+)
+@click.option(
+    "--trust",
+    is_flag=True,
+    help="Indicates the job bundle is trusted to not perform malicious activities",
```

Comment:
I think we can do better than an option like this. The question is about where to upload data from for jobs, not about which job bundles are ok or not.